src/wwwroot/locales/en-translation.js

```diff
@@ -511,5 +511,22 @@
   "resubscribe_internal_policies_error": "It is not possible to re-subscribe the email address due to conflicts with internal policies, please contact our support service.",
   "resubscribe_domain_error": "Verify that the domain is entered correctly and is configured in your account.",
   "resubscribe_unsubscription_error": "We cannot reactivate the Subscriber because it’s not inactive.",
-  "resubscribe_too_many_request_error": "You have reached your daily reactivation limit."
-};
+  "resubscribe_too_many_request_error": "You have reached your daily reactivation limit.",
+  "submenu_my_billing_information":"Billing Information",
+  "my_billing_information_text": "Billing Information",
+  "my_billing_information_subtitle": "The service will be charged to the credit card listed here. If you wish, you can <b>change the card.</b>",
+  "my_billing_information_change_credit_card": "Change Card",
+  "my_billing_information_back_button": "Back",
+  "my_billing_information_save_button": "Save",
+  "my_billing_information_payment_method_title": "Payment method",
+  "my_billing_information_credit_card_option": "Credit Card",
+  "my_billing_information_card_holder_label": "Cardholder",
```

Comment:
Is it okay to be like this "Cardholder"?, shouldn't  have space? 
```suggestion
  "my_billing_information_card_holder_label": "Card holder",

Reply:
yes, is ok: ```"Cardholder"```